app/models/message.rb

```diff
@@ -3,9 +3,19 @@
 
 # Message model
 class Message < Common::Base
+  MEGABYTE = 1024.0 * 1024.0
+  MAX_TOTAL_FILE_SIZE = 6.0
```

Comment:
I will look into making these changes retroactively in future PR since this got merged in already.